services/hanke-service/src/main/kotlin/fi/hel/haitaton/hanke/application/ApplicationService.kt

```diff
@@ -427,51 +427,69 @@ open class ApplicationService(
         }
     }
 
-    private fun sendApplicationToAllu(application: ApplicationEntity): Int {
+    private fun sendToAllu(entity: ApplicationEntity): Int {
+        val application = entity.toApplication()
+        val attachments = entity.attachments.map { it.toAlluAttachment() }
```

Comment:
The mapped attachments are only needed in the create-branch, so they should be mapped in that branch.

Reply:
Fixed as suggested.